ingestion/src/metadata/ingestion/source/database/databricks/metadata.py

```diff
@@ -157,20 +159,13 @@ def get_columns(self, connection, table_name, schema=None, **kw):
             "system_data_type": raw_col_type,
         }
         if col_type in {"array", "struct", "map"}:
-            if db_name and schema:
```

Comment:
the condition here had db_name and schema in the new only schema is there? Any reason for this change?

Reply:
Here `db_name` is not needed because at the start of ingestion we are using the `use db_name` query . I have discussed with @ulixius9 and then only removed and tested.